x/nft/bucket.go

```diff
@@ -12,6 +15,65 @@ type Owned interface {
 	OwnerAddress() weave.Address
 }
 
+//TODO: Better name?
+type BucketAccess interface {
+	Get(db weave.ReadOnlyKVStore, key []byte) (orm.Object, error)
+	Save(db weave.KVStore, model orm.Object) error
+}
+
+type BucketDispatcher interface {
+	Register(t string, bucket BucketAccess) error
+	AssertRegistered(types ...fmt.Stringer)
```

Comment:
I see why this is a Stringer from the code, but it would be more consistent to take a string (and just call `.String()` when used).

Reply:
it is a stringer, because we are using enums now.
sure, this can be a string too